build/r_snippets_menu_config.yml

```diff
@@ -1,4 +1,4 @@
-All of Us R snippets:
+All of Us SQL snippets:
```

Comment:
opt_nit: we may want to keep R and Py in the title. As a fallback, if I can't establish the language of the notebook for some reason (there aren't any known situations which produce this, but there may be some future edge case I have considered which triggers this situation), I'll just show both of the menus simultaneously. So it might be nice to disambiguate them in this situation.

Reply:
Updated.